notification/notification.go

```diff
@@ -0,0 +1,8 @@
+package notification
+
+import "github.com/flanksource/incident-commander/api"
+
+type INotifier interface {
```

Comment:
Just use `Notifier` the `I` prefix is not idiomatic

https://go.dev/doc/effective_go#interface-names